src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -42,251 +40,247 @@ const Content = ({
   releaseVersion,
   location,
   orgSlug,
+  activeDisplay,
   showPlaceholders,
 }: Props) => {
   const activeStatsPeriod = (location.query.healthStatsPeriod || '24h') as StatsPeriod;
-  const activeStatsSubject = (location.query.healthStat || 'sessions') as StatsSubject;
+  const healthStatsPeriod = (
+    <HealthStatsPeriod location={location} activePeriod={activeStatsPeriod} />
+  );
 
   return (
     <React.Fragment>
       <Header>
-        <HeaderLayout>
+        <Layout>
           <ProjectColumn>{t('Project name')}</ProjectColumn>
-          <AdoptionColumn>{t('Release adoption')}</AdoptionColumn>
-          <CrashFreeUsersColumn>{t('Crash free users')}</CrashFreeUsersColumn>
-          <CrashFreeSessionsColumn>{t('Crash free sessions')}</CrashFreeSessionsColumn>
-          <DailyUsersColumn>
-            <HealthStatsSubject location={location} activeSubject={activeStatsSubject} />
-            <HealthStatsPeriod location={location} activePeriod={activeStatsPeriod} />
-          </DailyUsersColumn>
+          <AdoptionColumn>{t('User Adoption')}</AdoptionColumn>
+          {activeDisplay === DisplayOption.CRASH_FREE_USERS ? (
+            <React.Fragment>
+              <UsersColumn>{t('Crash-free users')}</UsersColumn>
```

Comment:
We don't do `Crash-free users` with `-` in designs nor Display and Sort dropdowns. Should we stay consistent?